force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +287,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book testBook = new Book('A Brief History of Time', 'Stephen Hawking');
 
         // Call the getBookDetails method on the book object to get the details of the book
-
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return testBook.getBookDetails();
```

Comment:
great. If you want this could be all one line of code. 

    return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();